config/config.go

```diff
@@ -54,13 +54,14 @@ type Config struct {
 	InstanceID string `json:"instanceID,omitempty"`
 
 	// MetricsConfig specifies configuration for metrics emission
-	MetricsConfig PrometheusConfig `json:"metricsConfig,omitempty"`
+	MetricsConfig MetricsConfig `json:"metricsConfig,omitempty"`
 
 	// FeatureFlags for general/experimental features
 	FeatureFlags FeatureFlags `json:"featureFlags,omitempty"`
 
 	// TelemetryConfig specifies configuration for telemetry emission
-	TelemetryConfig PrometheusConfig `json:"telemetryConfig,omitempty"`
+	// DEPRECATED: Telemetry is emitted under metrics. See "metricsConfig"
+	TelemetryConfig MetricsConfig `json:"telemetryConfig,omitempty"`
```

Comment:
I'd like us to keep the telemetry endpoint - but allow them both on the same port/path. I think we can be backwards compatible here. Also - Darwin does not support the current telemetry  and returns a 500 error.

Reply:
Should telemetry be on by default? Should it use the same endpoint as metrics by default?

Reply:
I agree with this. On by default and same port.